src/edframe.cpp

```diff
@@ -1764,6 +1761,7 @@ void PoeditFrame::OnUpdateFromPOTUpdate(wxUpdateUIEvent& event)
 #ifdef HAVE_HTTP_CLIENT
 void PoeditFrame::OnUpdateFromCrowdin(wxCommandEvent&)
 {
+    m_syncing = true;
     DoIfCanDiscardCurrentDoc([=]{
         CrowdinSyncFile(this, m_catalog, [=](std::shared_ptr<Catalog> cat){
             m_catalog = cat;
```

Comment:
508e4f920401ff39466007adf66754b730118123
- Fixed
- Improved/refactored
  - Implementation
  - Comment

Sorry for bug and unclear description and implementation creating so much confusion yet. 
Now it should be much more clear, at least at it seems to me or otherwise please feel free to ask any more questions.